polling/infrequent/compose_greeting_activity.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'temporalio/activity'
+require_relative 'test_service'
+
+module InfrequentPolling
```

Comment:
```suggestion
module Polling
  module Infrequent
```

Same with other files here so we can keep the modules matching the dir structure